Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/fragments/SaveLoadStateFragment.java

```diff
@@ -0,0 +1,94 @@
+package org.dolphinemu.dolphinemu.fragments;
+
+import android.app.Fragment;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.util.SparseIntArray;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.GridLayout;
+
+import org.dolphinemu.dolphinemu.R;
+import org.dolphinemu.dolphinemu.activities.EmulationActivity;
+
+public final class SaveLoadStateFragment extends Fragment implements View.OnClickListener
+{
+	public enum SaveOrLoad
```

Comment:
this sound as a flag, why not a simple boolean?

Reply:
Technically a boolean would work, but I think the readability of the enum is much better - "does True mean save or load?". There shouldn't be any performance issues with an enum.

Reply:
you have a good point